mobile-widgets/qmlmanager.cpp

```diff
@@ -1436,14 +1434,9 @@ QStringList QMLManager::buddyInit() const
 	int i = 0;
 	for_each_dive (i, d) {
 		QString temp = d->buddy;
-		if (!temp.isEmpty() && !temp.contains(",")){
+		if (!temp.isEmpty()){
 			buddies << d->buddy;
 		}
```

Comment:
That char * get converted into a QString and when you have a QString with ", " in it, and add that to a QStringList it apparently get parsed as a list. Or some other magic happens as I did not see any list entries with more than one name on it.
But since the existing code works it can stay.